src/components/views/auth/AuthBody.tsx

```diff
@@ -17,6 +17,16 @@ limitations under the License.
 import React from 'react';
 
 export default class AuthBody extends React.PureComponent {
+    componentDidMount() {
+        const unloadCallback = (event) => {
+            event.preventDefault();
+            event.returnValue = "";
+            return "";
+        };
+        window.addEventListener("beforeunload", unloadCallback);
+        return () => window.removeEventListener("beforeunload", unloadCallback);
```

Comment:
Returning a cleanup callback like this only works in React hooks, whereas this is a class component, so cleanup needs to go in `componentWillUnmount` instead.

Reply:
> Returning a cleanup callback like this only works in React hooks, whereas this is a class component, so cleanup needs to go in `componentWillUnmount` instead.

Thanks, i was going to correct this.